dateutil/tz/tz.py

```diff
@@ -1404,6 +1432,43 @@ def __get_gettz():
         tzlocal_classes += (tzwinlocal,)
 
     class GettzFunc(object):
+        """
+        gettz is designed not to be a public function. It is defined in a closure as a private function and called once. 
```

Comment:
This is not true. `GettzFunc` is not a public function. `gettz` is very much a public function, otherwise it wouldn't be in `__all__` and wouldn't be getting any documentation.

Reply:
Ah sorry about that. I misunderstood your other comment in the original issue.